Java/pom.xml

```diff
@@ -408,6 +415,13 @@
       <version>${joda.version}</version>
     </dependency>
 
+    <!-- Adds a dependency on the Hadoop Core module. -->
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-core</artifactId>
+      <version>1.2.1</version>
```

Comment:
This is a very old version of hadoop-client, which was then moved to org.apache.hadoop:hadoop-client. Can we use hadoop 3 or [2.10.2](https://mvnrepository.com/artifact/org.apache.hadoop/hadoop-client/2.10.2)?